services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling the `DoesNotExist` exception that may be raised by `MovieSession.objects.get` to provide a more user-friendly error message or behavior.